discovery/manager.go

```diff
@@ -428,7 +428,7 @@ func (m *Manager) registerProviders(cfgs Configs, setName string) int {
 		}
 		typ := cfg.Name()
 		d, err := cfg.NewDiscoverer(DiscovererOptions{
-			Logger:            log.With(m.logger, "discovery", typ),
+			Logger:            log.With(m.logger, "discovery", typ, "set", setName),
```

Comment:
Might "config" be a more clear name? As you point out in the PR description, "set" is more neutral as the value comes from both scrape "jobs"  and Alertmanager "configs," but are those not both SD "configs" in the end? Also, outside of the context of the codebase I find "set" to be quite meaningless, and I wonder if any normal user could discern what it means seeing it in a log message.

Reply:
Thanks; I changed it to "config".